src/components/config/validate.ts

```diff
@@ -304,12 +304,12 @@ export const validateConfig = (configuration: Config): Validation => {
   }
 
   // Validate probes
-  if ((configuration?.probes?.length ?? 0) === 0) return NO_PROBES
+  if (probes.length === 0) return NO_PROBES
 
   // Check probes properties
   for (const probe of probes) {
-    const { alerts = [], requests, socket } = probe
-    const socketAlerts = probe.socket?.alerts ?? []
+    const { name, interval, alerts = [], requests, socket } = probe
```

Comment:
I think we also need to validate the interval to be greater than 0

Reply:
Updated!